clr-k8s-examples/vagrant.md

```diff
@@ -36,18 +36,34 @@ Note, vagrant installation steps were derived from:
 
 On Clear Linux, run these commands
 ```bash
-sudo wget https://raw.githubusercontent.com/AntonioMeireles/ClearLinux-packer/master/extras/clearlinux/setup/libvirtd.sh
+wget https://raw.githubusercontent.com/AntonioMeireles/ClearLinux-packer/master/extras/clearlinux/setup/libvirtd.sh
 chmod +x libvirtd.sh
 ./libvirtd.sh
-sudo wget https://raw.githubusercontent.com/AntonioMeireles/ClearLinux-packer/master/extras/clearlinux/setup/vagrant.sh
+```
```

Comment:
Can we make dependencies the first task so it is consistent with instructions for other OS?

Reply:
Yes, will do @jascott1